### PR TITLE
Balanced Frezon price

### PR DESCRIPTION
**Pull request by 778b**
## About the PR 

**Media**

Content Client_VnY09VTEJj

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

Tritium isnt change because it have too low price per full canister (not imbalance)
Frezon had change price per mole and full canister of Frezon now cost around 54.000 spacebux (was around 174.000 spacebux)

Closes: https://github.com/space-wizards/space-station-14/issues/11286

:cl:778b
- fix: Balanced Frezon price per mole

### CODE REVIEW

**Comment by asperger-sind** [comment]
tl;dr
frezon synthesis takes too much time and effort to even make it as a reasonable income source, to that you have to actually get competent atmos techs to synthesize it for the duration of nearly the entire round

Frezon should keep it's price for how hard it is to synthesize a reasonable amount of it in the span of a round, maybe even bump it up higher.

To actually get at the very least 500 mols of frezon (now worth a grand total of $1500) you need:

Roundstart atmos gear-up, setup, etc. - 2~ minutes
Rejig of atmos for space and steel - 7~ minutes
Rejig of the mix chamber for optimal tritium synthesis - up to 20 mins, air alarm settings are still bullshit

500 mols of tritium - another 20~ mins even with an efficient setup (less could be require actually, not exactly sure on the frezon production reaction details)

A frezon synthesis loop - 5~ mins
Frezon production - Rest of the round length

For a grand total of 50+ minutes of peak performance atmos tech gameplay JUST for the setup. 
By this point the round can be over, and you just end up completely ignoring your other duties, and even if you make the frezon in the amount of 500 mols, you still only get a canister worth $2500 (1000 for the canister itself!), a price cargo could fetch from you if you just can them miner gasses at low mole concentration.

**Comment by EmoGarbage404** [comment]
I'm just going to comment that:
1. This is much more effective the longer the round lasts. A 2 to 3 hour round could easily make heavy use of this production.
2. Gas prices in general shouldn't be that high, considering gases are infinite in the scope of atmos and only require a preliminary setup.

**Comment by 778b** [comment]
> tl;dr frezon synthesis takes too much time and effort to even make it as a reasonable income source, to that you have to actually get competent atmos techs to synthesize it for the duration of nearly the entire round
> 
> Frezon should keep it's price for how hard it is to synthesize a reasonable amount of it in the span of a round, maybe even bump it up higher.
> 
> To actually get at the very least 500 mols of frezon (now worth a grand total of $1500) you need:
> 
> Roundstart atmos gear-up, setup, etc. - 2~ minutes Rejig of atmos for space and steel - 7~ minutes Rejig of the mix chamber for optimal tritium synthesis - up to 20 mins, air alarm settings are still bullshit
> 
> 500 mols of tritium - another 20~ mins even with an efficient setup (less could be require actually, not exactly sure on the frezon production reaction details)
> 
> A frezon synthesis loop - 5~ mins Frezon production - Rest of the round length
> 
> For a grand total of 50+ minutes of peak performance atmos tech gameplay JUST for the setup. By this point the round can be over, and you just end up completely ignoring your other duties, and even if you make the frezon in the amount of 500 mols, you still only get a canister worth $2500 (1000 for the canister itself!), a price cargo could fetch from you if you just can them miner gasses at low mole concentration.
Usually u had 3 free canister in round begin, and if ur game longer than 1+ hour or u already had tritium leak event, or tritium artefact, u will just had FREE FREZON, and in ss13 game usually longer than 1-2 hours, and sometimes in ss14 i do more than 300.000$ in one 2 hours game, its not a imbalance u think?

**Comment by asperger-sind** [comment]
> Usually u had 3 free canister in round begin, and if ur game longer than 1+ hour or u already had tritium leak event, or tritium artefact, u will just had FREE FREZON, and in ss13 game usually longer than 1-2 hours, and sometimes in ss14 i do more than 300.000$ in one 2 hours game, its not a imbalance u think?

1. No map on upstream has 3 frezon canisters roundstart, Mirror agreed that there AT MOST should be one mapped into a station.

2. Leak events do not make nearly as much as you need, and it either burns away or gets spaced, recycling the tritium efficiently is a pain as well.

3. Any form of tritium generating artifact is already like an infinite cash hack, you don't need to even synthesize it into frezon.

4. You don't RELY on Atmos for money, it's not like the station burns down and crashes without an atmos tech making that big bucks for cargo, said cargo with salvage or in cooperation with science can make more money than cargo can reasonably use.

**Comment by asperger-sind** [comment]
> I'm just going to comment that:
> 
> 1. This is much more effective the longer the round lasts. A 2 to 3 hour round could easily make heavy use of this production.
> 2. Gas prices in general shouldn't be that high, considering gases are infinite in the scope of atmos and only require a preliminary setup.

1. In a 2-3 hour long round a competent salvage/cargo crew is going to make more money than they can reasonably spend, even without help from other crewmembers, and considering that only Salamander can really achieve rounds as long as these you need even less money to sustain a skeleton crew.

2. I don't see an issue with the hardest gas in the game to synthesize being worthy of selling, besides, gas miners gotta go at some point.

I've also missed out this on the PR;
Frezon being cheaper than tritium which it's made out of is plain baffling to me, and the reasoning of tritium being harder to store in overly large quantities than frezon thus making it balanced is odd to me.
I could reason this with the conversion reaction at peak efficiency making you more frezon moles than you spend tritium moles, but again, not exactly sure on the peaks of the production reaction.


either way this stupid discussion should probably happen on discord and not on github, probably with mirror present as they made frezon

**Comment by Scribbles0** [comment]
just gonna add that this doesn't happen as often as one might think, salamander doesn't do it much and i don't think lizard rounds last long enough to make enough frezon to get a reasonable amount of money anyway.
(also there are _mapped_ frezon canisters?)

**Comment by 778b** [comment]
> > Usually u had 3 free canister in round begin, and if ur game longer than 1+ hour or u already had tritium leak event, or tritium artefact, u will just had FREE FREZON, and in ss13 game usually longer than 1-2 hours, and sometimes in ss14 i do more than 300.000$ in one 2 hours game, its not a imbalance u think?
> 
> 1. No map on upstream has 3 frezon canisters roundstart, Mirror agreed that there AT MOST should be one mapped into a station.
> 2. Leak events do not make nearly as much as you need, and it either burns away or gets spaced, recycling the tritium efficiently is a pain as well.
> 3. Any form of tritium generating artifact is already like an infinite cash hack, you don't need to even synthesize it into frezon.
> 4. You don't RELY on Atmos for money, it's not like the station burns down and crashes without an atmos tech making that big bucks for cargo, said cargo with salvage or in cooperation with science can make more money than cargo can reasonably use.

I dont say 3 free canister of Frezon, im said about empty 3 free canister (cuz u said about price of empty canister in first comment), and IMHO Frezon is imbalance, I usually use Atmo to get some 200.000 spacebux, and i found issue about it, and fix it. Its fact, i cant do 300.000 spacebux on cluster map with 1 scientist(With powergame or with access to lab i can do without scientist) and 1 amto engineer for 2 hours like easy, for 1.5 hour do full canister of Frezon which cost 170.000 spacebux

**Comment by GoodWheatley** [comment]
> > > Usually u had 3 free canister in round begin, and if ur game longer than 1+ hour or u already had tritium leak event, or tritium artefact, u will just had FREE FREZON, and in ss13 game usually longer than 1-2 hours, and sometimes in ss14 i do more than 300.000$ in one 2 hours game, its not a imbalance u think?
> > 
> > 
> > 
> > 1. No map on upstream has 3 frezon canisters roundstart, Mirror agreed that there AT MOST should be one mapped into a station.
> > 2. Leak events do not make nearly as much as you need, and it either burns away or gets spaced, recycling the tritium efficiently is a pain as well.
> > 3. Any form of tritium generating artifact is already like an infinite cash hack, you don't need to even synthesize it into frezon.
> > 4. You don't RELY on Atmos for money, it's not like the station burns down and crashes without an atmos tech making that big bucks for cargo, said cargo with salvage or in cooperation with science can make more money than cargo can reasonably use.
> 
> I dont say 3 free canister of Frezon, im said about empty 3 free canister (cuz u said about price of empty canister in first comment), and IMHO Frezon is imbalance, I usually use Atmo to get some 200.000 spacebux, and i found issue about it, and fix it. Its fact, i cant do 300.000 spacebux on cluster map with 1 scientist(With powergame or with access to lab i can do without scientist) and 1 amto engineer for 2 hours like easy, for 1.5 hour do full canister of Frezon which cost 170.000 spacebux

1. Nobody ever mentioned the price of an empty canister. You brought it up first by saying maps had 3 frezon canisters mapped in.
2. You need to understand how the atmos system works to make frezon. You know how the system works, ergo making frezon isn’t hard for you and you can mass-produce it for cash. 95% of atmos techs struggle to just make tritium, let alone frezon.
3. Tritium is extremely hard to produce in large quantities, meaning it’s a bottleneck for frezon production. Unless you know the optimal way to produce tritium and spend all your time setting up the pipes to start production, it will take almost the entire round to produce even a canister of it, meaning making frezon is impossible to do for most atmos techs.

Overall, I don’t think the price of frezon is an issue right now.

**Comment by 778b** [comment]
[asperger-sind](https://github.com/asperger-sind) said "...you still only get a canister worth $2500 (1000 for the canister itself!)...", he/she talk about empty canister cost, and i answer him/her, about on any map u had 3 free empty canister.
 And its possible to create full canister of Frezon for 1-2 hours game for easy

**Comment by moonheart08** [comment]
Frezon balance is fine where it is, production becomes problematic only in rounds notably longer than 2hr, which is above the mark upstream is balanced for.

**Comment by UbaserB** [comment]
huh, maybe we should revive this

**Comment by deltanedas** [comment]
on deltav i had a pro speedrun frezon and made a can in 28 mins (while i speedran ame 28 seconds :trollface:) so its absolutely possible if you are good
